QHyper/problems/maxcut.py

```diff
@@ -1,27 +1,31 @@
-import random
-import sympy
-from collections import namedtuple
-
 from typing import cast
 
-from .base import Problem
+import sympy
 from QHyper.hyperparameter_gen.parser import Expression
 
+from .base import Problem
+
 
 class MaxCutProblem(Problem):
     def __init__(self, edges: list[tuple[int, int]]) -> None:
         self.edges = edges
-        self.variables = sympy.symbols(' '.join(
-            [f'x{i}' for i in range(max(v for edge in edges for v in edge) + 1)]
-        ))
+        self.variables = sympy.symbols(
+            " ".join(
+                [
+                    f"x{i}"
+                    for i in range(max(v for edge in edges for v in edge))
+                ]
+            )
+        )
```

Comment:
To be fair, I'm not a fan of this version which occupies 8 lines

Reply:
Oops black got too rigid